CHANGELOG.md

```diff
@@ -3,8 +3,18 @@
 ## Next
 
 ### Added
-- Introduced optional lexical graph configuration for SimpleKGPipeline, enhancing flexibility in customizing node labels and relationship types in the lexical graph.
+- Introduced optional lexical graph configuration for `SimpleKGPipeline`, enhancing flexibility in customizing node labels and relationship types in the lexical graph.
+- Introduced optional `neo4j_database` parameter for `SimpleKGPipeline` to create the graph in a non-default database.
+- Introduced optional `neo4j_database` parameter for `Neo4jChunkReader` to read chunks from a non-default database.
+- Introduced optional `neo4j_database` parameter for `Text2CypherRetriever` to run the generated query against a non-default database.
 - Ability to provide description and list of properties for entities and relations in the SimpleKGPipeline constructor.
+- Ability to run queries against a custom `neo4j_database` for the `Text2CypherRetriever` and the `Neo4jChunkReader` component (was already possible with other retrievers and components).
```

Comment:
It's a bit nit-picky but I feel like there's a bit of redundancy in this change log update. Maybe combine a few of these lines?

Reply:
Agreed, it was a bit weird... Updated in https://github.com/neo4j/neo4j-graphrag-python/pull/216/commits/80718ee38fbda1cf4e147e4c1a50eedfb12e54bc